src/main/scala/org/bblfsh/client/BblfshClient.scala

```diff
@@ -30,13 +29,15 @@ class BblfshClient(host: String, port: Int, maxMsgSize: Int) {
     parsed
   }
 
-  def filter(node: Node, query: String): List[Node] = Libuast.synchronized {
-    libuast.filter(node, query)
+  // proxy to the static method for backward compatibility
```

Comment:
Maybe better to use [scaladoc style](https://docs.scala-lang.org/style/scaladoc.html)?